qa/pull-tester/rpc-tests.sh

```diff
@@ -83,7 +83,8 @@ testScripts=(
     'turnstile.py'
     'mining_shielded_coinbase.py'
     'framework.py'
-    'feature_zip221.py'
+    'feature_zip221.py',
```

Comment:
Alter this commit to remove the trailing comma (this file is a Python script in the upstream commit, but still a Bash script here).